shared/version/api.go

```diff
@@ -368,6 +368,7 @@ var APIExtensions = []string{
 	"migration_vm_live",
 	"ovn_nic_nesting",
 	"oidc",
+	"ovn_nic_acceleration_vdpa",
```

Comment:
This commit should be include the changes to [doc/api-extensions.md](https://github.com/lxc/lxd/pull/11528/commits/556eda5c2e22150880aa8e063bb0611ba4527a66#diff-906870407147ce5fd1c4850626a492df3cd564c6981706e8ef8f7247e9200637) and the commit message should be:

`api: Add the ovn_nic_acceleration_vdpa extension`